docs/team/nazrylLim.md

```diff
@@ -8,19 +8,69 @@ NotUS is a quick and simple, Command Line Interface (CLI) based, note-taking app
 
 *Roles (short for ‘in charge of role’): Description*
 
-- 
+- Architecture: Worked with Jun Xiang on setting up the project, planned on logic of the project structure. Provides coding standards for programmers to follow.
+- In charge of `add-n`,`delete-n`, `edit-n` of the code.
 
 ## Summary of Contributions
 
 ### Features implemented
+
 1. Set up the skeleton code used in the project, including the following classes:
+    * `Notebook` Class
+    * `Note` Class
     * `InterfaceManager` Class
+    * `SystemException` Class
+    * `PrefixSyntax` Class
+    
+1. Completed the following features and test code for
+    * `add-n`
+    * `delete-n`
+    * `edit-n`
```

Comment:
I haven't merged yet, you can look at my PR but I think we need to give a brief explanation of what each feature does, why, and how it helps the user.